Bottom: ce5b1e9f412916766d04d7a2a720dcea2879fb87
Top:    e83a3ba5ad73d6644c39499779563379ef136fad
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-01-24 17:49:39 +0000

Refresh of drop-in-pcb-gl-code-various-me

---

diff --git a/doc/actions.texi b/doc/actions.texi
index cdc1a71..f8c4261 100644
--- a/doc/actions.texi
+++ b/doc/actions.texi
@@ -1,5 +1,5 @@
 @c key actions
-@c ./../src/action.c 212
+@c ./../src/action.c 211
 
 Many actions take a @code{delta} parameter as the last parameter,
 which is an amount to change something.  That @code{delta} may include
@@ -18,7 +18,7 @@ Action(Object,-1)
 Actions which take a @code{delta} parameter which do not accept all
 these options will specify what they do take.
 
-@c ./../src/action.c 216
+@c ./../src/action.c 215
 
 @macro pinshapes
 
@@ -46,7 +46,7 @@ is documented for that purpose.
 
 @end macro
 
-@c ./../src/action.c 214
+@c ./../src/action.c 213
 
 Many actions act on indicated objects on the board.  They will have
 parameters like @code{ToggleObject} or @code{SelectedVias} to indicate
@@ -128,7 +128,6 @@ Affects all objects which are both selected and of the @var{Type} specified.
 * m Action:: Loads a layout into the current buffer.
 * MarkCrosshair Action:: Set/Reset the Crosshair mark
 * Message Action:: Writes a message to the log window.
-* MinClearGap Action:: Ensures that polygons are a minimum distance from objects.
 * MinMaskGap Action:: Ensures the mask is a minimum distance from pins and pads.
 * Mode Action:: Change or use the tool mode.
 * MorphPolygon Action:: Converts dead polygon islands into separate polygons.
@@ -186,7 +185,7 @@ AddRats(AllRats|SelectedRats|Close)@end format
 @end cartouche
 
 Add one or more rat lines to the board.
-@c ./../src/action.c 3203
+@c ./../src/action.c 3194
 
 @table @code
 
@@ -230,7 +229,7 @@ Atomic(Save|Restore|Close|Block)@end format
 @end cartouche
 
 Save or restore the undo serial number.
-@c ./../src/action.c 1607
+@c ./../src/action.c 1605
 
 This action allows making multiple-action bindings into an atomic
 operation that will be undone by a single Undo command.  For example,
@@ -270,7 +269,7 @@ AutoPlaceSelected()@end format
 @end cartouche
 
 Auto-place selected components.
-@c ./../src/action.c 3350
+@c ./../src/action.c 3341
 
 Attempts to re-arrange the selected components such that the nets
 connecting them are minimized.  Note that you cannot undo this.
@@ -285,7 +284,7 @@ AutoRoute(AllRats|SelectedRats)@end format
 @end cartouche
 
 Auto-route some or all rat lines.
-@c ./../src/action.c 3373
+@c ./../src/action.c 3364
 
 @table @code
 
@@ -319,7 +318,7 @@ ChangeClearSize(Selected|SelectedObjects, delta)@end format
 @end cartouche
 
 Changes the clearance size of objects.
-@c ./../src/action.c 3591
+@c ./../src/action.c 3582
 
 If the solder mask is currently showing, this action changes the
 solder mask clearance.  If the mask is not showing, this action
@@ -336,7 +335,7 @@ ChangeDrillSize(SelectedPins|SelectedVias|Selected|SelectedObjects, delta)@end f
 @end cartouche
 
 Changes the drilling hole size of objects.
-@c ./../src/action.c 3533
+@c ./../src/action.c 3524
 
 
 @node ChangeFlag Action
@@ -353,7 +352,7 @@ flag = square | octagon | thermal | join
 @end cartouche
 
 Sets or clears flags on objects.
-@c ./../src/action.c 5668
+@c ./../src/action.c 5561
 
 Toggles the given flag on the indicated object(s).  The flag may be
 one of the flags listed above (square, octagon, thermal, join).  The
@@ -370,7 +369,7 @@ ChangeHole(ToggleObject|Object|SelectedVias|Selected)@end format
 @end cartouche
 
 Changes the hole flag of objects.
-@c ./../src/action.c 4492
+@c ./../src/action.c 4385
 
 The "hole flag" of a via determines whether the via is a
 plated-through hole (not set), or an unplated hole (set).
@@ -385,7 +384,7 @@ ChangeJoin(ToggleObject|SelectedLines|SelectedArcs|Selected)@end format
 @end cartouche
 
 Changes the join (clearance through polygons) of objects.
-@c ./../src/action.c 4106
+@c ./../src/action.c 3999
 
 The join flag determines whether a line or arc, drawn to intersect a
 polygon, electrically connects to the polygon or not.  When joined,
@@ -404,7 +403,7 @@ ChangeName(Object)
 @end cartouche
 
 Sets the name of objects.
-@c ./../src/action.c 3909
+@c ./../src/action.c 3802
 
 @table @code
 
@@ -430,7 +429,7 @@ ChangeOctagon(SelectedElements|SelectedPins|SelectedVias)@end format
 @end cartouche
 
 Changes the octagon-flag of pins and vias.
-@c ./../src/action.c 4318
+@c ./../src/action.c 4211
 
 @pinshapes
 
@@ -444,7 +443,7 @@ ChangePaste(ToggleObject|Object|SelectedPads|Selected)@end format
 @end cartouche
 
 Changes the no paste flag of objects.
-@c ./../src/action.c 4534
+@c ./../src/action.c 4427
 
 The "no paste flag" of a pad determines whether the solderpaste
  stencil will have an opening for the pad (no set) or if there wil be
@@ -461,7 +460,7 @@ ChangePinName(ElementName,PinNumber,PinName)@end format
 @end cartouche
 
 Sets the name of a specific pin on a specific element.
-@c ./../src/action.c 3833
+@c ./../src/action.c 3726
 
 This can be especially useful for annotating pin names from a
 schematic to the layout without requiring knowledge of the pcb file
@@ -485,7 +484,7 @@ ChangeSize(SelectedElements, delta)@end format
 @end cartouche
 
 Changes the size of objects.
-@c ./../src/action.c 3444
+@c ./../src/action.c 3435
 
 For lines and arcs, this changes the width.  For pins and vias, this
 changes the overall diameter of the copper annulus.  For pads, this
@@ -505,7 +504,7 @@ ChangeSquare(Selected|SelectedObjects)@end format
 @end cartouche
 
 Changes the square flag of pins and pads.
-@c ./../src/action.c 4159
+@c ./../src/action.c 4052
 
 Note that @code{Pins} means both pins and pads.
 
@@ -522,7 +521,7 @@ ClearOctagon(SelectedElements|SelectedPins|SelectedVias)@end format
 @end cartouche
 
 Clears the octagon-flag of pins and vias.
-@c ./../src/action.c 4434
+@c ./../src/action.c 4327
 
 @pinshapes
 
@@ -536,7 +535,7 @@ ClearSquare(ToggleObject|SelectedElements|SelectedPins)@end format
 @end cartouche
 
 Clears the square-flag of pins and pads.
-@c ./../src/action.c 4265
+@c ./../src/action.c 4158
 
 Note that @code{Pins} means pins and pads.
 
@@ -556,7 +555,7 @@ flag = square | octagon | thermal | join@end format
 @end cartouche
 
 Clears flags on objects.
-@c ./../src/action.c 5651
+@c ./../src/action.c 5544
 
 Turns the given flag off, regardless of its previous setting.  See
 @code{ChangeFlag}.
@@ -575,7 +574,7 @@ Connection(Find|ResetLinesAndPolygons|ResetPinsAndVias|Reset)@end format
 @end cartouche
 
 Searches connections of the object at the cursor position.
-@c ./../src/action.c 2027
+@c ./../src/action.c 2025
 
 Connections found with this action will be highlighted in the
 ``connected-color'' color and will have the ``found'' flag set.
@@ -611,7 +610,7 @@ Delete(AllRats|SelectedRats)  ;@end format
 @end cartouche
 
 Delete stuff.
-@c ./../src/action.c 3267
+@c ./../src/action.c 3258
 
 
 @node DeleteRats Action
@@ -623,7 +622,7 @@ DeleteRats(AllRats|Selected|SelectedRats)@end format
 @end cartouche
 
 Delete rat lines.
-@c ./../src/action.c 3316
+@c ./../src/action.c 3307
 
 
 @node DisableVendor Action
@@ -655,7 +654,7 @@ DisperseElements(All|Selected)@end format
 @end cartouche
 
 Disperses elements.
-@c ./../src/action.c 2073
+@c ./../src/action.c 2071
 
 Normally this is used when starting a board, by selecting all elements
 and then dispersing them.  This scatters the elements around the board
@@ -682,7 +681,7 @@ Display(Pinout|PinOrPadName)
 @end cartouche
 
 Several display-related actions.
-@c ./../src/action.c 2191
+@c ./../src/action.c 2189
 
 @table @code
 
@@ -868,7 +867,7 @@ DRC()@end format
 @end cartouche
 
 Invoke the DRC check.
-@c ./../src/action.c 1642
+@c ./../src/action.c 1640
 
 Note that the design rule check uses the current board rule settings,
 not the current style settings.
@@ -883,7 +882,7 @@ DumpLibrary()@end format
 @end cartouche
 
 Display the entire contents of the libraries.
-@c ./../src/action.c 1676
+@c ./../src/action.c 1674
 
 
 
@@ -918,7 +917,7 @@ ExecuteFile(filename)@end format
 @end cartouche
 
 Run actions from the given file.
-@c ./../src/action.c 5782
+@c ./../src/action.c 5675
 
 Lines starting with @code{#} are ignored.
 
@@ -932,7 +931,7 @@ Flip(Object|Selected|SelectedElements)@end format
 @end cartouche
 
 Flip an element to the opposite side of the board.
-@c ./../src/action.c 1726
+@c ./../src/action.c 1724
 
 Note that the location of the element will be symmetric about the
 cursor location; i.e. if the part you are pointing at will still be at
@@ -1038,7 +1037,7 @@ LoadFrom(Layout|LayoutToBuffer|ElementToBuffer|Netlist|Revert,filename)@end form
 @end cartouche
 
 Load layout data from a file.
-@c ./../src/action.c 4946
+@c ./../src/action.c 4839
 
 This action assumes you know what the filename is.  The various GUIs
 should have a similar @code{Load} action where the filename is
@@ -1118,7 +1117,7 @@ MarkCrosshair()
 @end cartouche
 
 Set/Reset the Crosshair mark
-@c ./../src/action.c 3406
+@c ./../src/action.c 3397
 
 The ``mark'' is a small X-shaped target on the display which is
 treated like a second origin (the normal origin is the upper let
@@ -1140,7 +1139,7 @@ Message(message)@end format
 @end cartouche
 
 Writes a message to the log window.
-@c ./../src/action.c 1773
+@c ./../src/action.c 1771
 
 This action displays a message to the log window.  This action is primarily
 provided for use by other programs which may interface with PCB.  If
@@ -1148,23 +1147,6 @@ multiple arguments are given, each one is sent to the log window
 followed by a newline.
 
 
-@node MinClearGap Action
-@subsection MinClearGap
-@c key MinClearGap in hid 
-@cartouche
-@format
-MinClearGap(delta)
-"``MinClearGap(Selected, delta)@end format
-@end cartouche
-
-Ensures that polygons are a minimum distance from objects.
-@c ./../src/action.c 3735
-
-Checks all specified objects, and increases the polygon clearance if
-needed to ensure a minimum distance between their edges and the
-polygon edges.
-
-
 @node MinMaskGap Action
 @subsection MinMaskGap
 @c key MinMaskGap in hid 
@@ -1175,7 +1157,7 @@ MinMaskGap(delta)
 @end cartouche
 
 Ensures the mask is a minimum distance from pins and pads.
-@c ./../src/action.c 3659
+@c ./../src/action.c 3650
 
 Checks all specified pins and/or pads, and increases the mask if
 needed to ensure a minimum distance between the pin or pad edge and
@@ -1194,7 +1176,7 @@ Mode(Notify|Release|Cancel|Stroke)
 @end cartouche
 
 Change or use the tool mode.
-@c ./../src/action.c 2522
+@c ./../src/action.c 2513
 
 @table @code
 
@@ -1254,7 +1236,7 @@ MorphPolygon(Object|Selected)@end format
 @end cartouche
 
 Converts dead polygon islands into separate polygons.
-@c ./../src/action.c 3991
+@c ./../src/action.c 3884
 
 If a polygon is divided into unconnected "islands", you can use
 this command to convert the otherwise disappeared islands into
@@ -1322,7 +1304,7 @@ MoveObject(X,Y,dim)@end format
 @end cartouche
 
 Moves the object under the crosshair.
-@c ./../src/action.c 5483
+@c ./../src/action.c 5376
 
 The @code{X} and @code{Y} are treated like @code{delta} is for many
 other objects.  For each, if it's prefixed by @code{+} or @code{-},
@@ -1340,7 +1322,7 @@ MoveToCurrentLayer(Object|SelectedObjects)@end format
 @end cartouche
 
 Moves objects to the current layer.
-@c ./../src/action.c 5525
+@c ./../src/action.c 5418
 
 Note that moving an element from a component layer to a solder layer,
 or from solder to component, won't automatically flip it.  Use the
@@ -1396,7 +1378,7 @@ New([name])@end format
 @end cartouche
 
 Starts a new layout.
-@c ./../src/action.c 5009
+@c ./../src/action.c 4902
 
 If a name is not given, one is prompted for.
 
@@ -1433,7 +1415,7 @@ PasteBuffer(ToLayout, X, Y, units)@end format
 @end cartouche
 
 Various operations on the paste buffer.
-@c ./../src/action.c 5073
+@c ./../src/action.c 4966
 
 There are a number of paste buffers; the actual limit is a
 compile-time constant @code{MAX_BUFFER} in @file{globalconst.h}.  It
@@ -1492,7 +1474,7 @@ Polygon(Close|PreviousPoint)@end format
 @end cartouche
 
 Some polygon related stuff.
-@c ./../src/action.c 5419
+@c ./../src/action.c 5312
 
 Polygons need a special action routine to make life easier.
 
@@ -1576,7 +1558,7 @@ Quit()@end format
 @end cartouche
 
 Quits the application after confirming.
-@c ./../src/action.c 2005
+@c ./../src/action.c 2003
 
 If you have unsaved changes, you will be prompted to confirm (or
 save) before quitting.
@@ -1591,7 +1573,7 @@ Redo()@end format
 @end cartouche
 
 Redo recent``undo''operations.
-@c ./../src/action.c 5385
+@c ./../src/action.c 5278
 
 This routine allows you to recover from the last undo command.  You
 might want to do this if you thought that undo was going to revert
@@ -1614,7 +1596,7 @@ RemoveSelected()@end format
 @end cartouche
 
 Removes any selected objects.
-@c ./../src/action.c 2729
+@c ./../src/action.c 2720
 
 
 @node Renumber Action
@@ -1628,7 +1610,7 @@ Renumber()
 
 Renumber all elements.  The changes will be recorded to filename
 for use in backannotating these changes to the schematic.
-@c ./../src/action.c 2747
+@c ./../src/action.c 2738
 
 
 @node Report Action
@@ -1686,7 +1668,7 @@ RipUp(All|Selected|Element)@end format
 @end cartouche
 
 Ripup auto-routed tracks, or convert an element to parts.
-@c ./../src/action.c 3099
+@c ./../src/action.c 3090
 
 @table @code
 
@@ -1735,7 +1717,7 @@ RouteStyle(1|2|3|4)@end format
 @end cartouche
 
 Copies the indicated routing style into the current sizes.
-@c ./../src/action.c 5451
+@c ./../src/action.c 5344
 
 
 @node s Action
@@ -1767,7 +1749,7 @@ SaveSettings()
 @end cartouche
 
 Saves settings.
-@c ./../src/action.c 4930
+@c ./../src/action.c 4823
 
 If you pass no arguments, the settings are stored in
 @code{$HOME/.pcb/settings}.  If you pass the word @code{local} they're
@@ -1785,7 +1767,7 @@ SaveTo(PasteBuffer,filename)@end format
 @end cartouche
 
 Saves data to a file.
-@c ./../src/action.c 4840
+@c ./../src/action.c 4733
 
 @table @code
 
@@ -1825,7 +1807,7 @@ Select(TextByName|ViaByName, Name)
 @end cartouche
 
 Toggles or sets the selection
-@c ./../src/action.c 4576
+@c ./../src/action.c 4469
 
 @table @code
 
@@ -1874,7 +1856,7 @@ flag = square | octagon | thermal | join@end format
 @end cartouche
 
 Sets flags on objects.
-@c ./../src/action.c 5634
+@c ./../src/action.c 5527
 
 Turns the given flag on, regardless of its previous setting.  See
 @code{ChangeFlag}.
@@ -1893,7 +1875,7 @@ SetOctagon(Object|ToggleObject|SelectedElements|Selected)@end format
 @end cartouche
 
 Sets the octagon-flag of objects.
-@c ./../src/action.c 4376
+@c ./../src/action.c 4269
 
 @pinshapes
 
@@ -1907,7 +1889,7 @@ SetSame()@end format
 @end cartouche
 
 Sets current layer and sizes to match indicated item.
-@c ./../src/action.c 5566
+@c ./../src/action.c 5459
 
 When invoked over any line, arc, polygon, or via, this changes the
 current layer to be the layer that item is on, and changes the current
@@ -1923,7 +1905,7 @@ SetSquare(ToggleObject|SelectedElements|SelectedPins)@end format
 @end cartouche
 
 sets the square-flag of objects.
-@c ./../src/action.c 4212
+@c ./../src/action.c 4105
 
 Note that @code{Pins} means pins and pads.
 
@@ -1945,7 +1927,7 @@ Style = 2 has horizontal and vertical fingers with sharp edges.
 Style = 3 is a solid connection to the plane.Style = 4 has diagonal fingers with rounded edges.
 Style = 5 has horizontal and vertical fingers with rounded edges.
 
-@c ./../src/action.c 1799
+@c ./../src/action.c 1797
 
 This changes how/whether pins or vias connect to any rectangle or polygon
 on the current layer. The first argument can specify one object, or all
@@ -1971,7 +1953,7 @@ SetValue(Grid|Line|LineSize|Text|TextScale|ViaDrillingHole|Via|ViaSize, delta)@e
 @end cartouche
 
 Change various board-wide values and sizes.
-@c ./../src/action.c 1918
+@c ./../src/action.c 1916
 
 @table @code
 
@@ -2005,7 +1987,7 @@ ToggleHideName(Object|SelectedElements)@end format
 @end cartouche
 
 Toggles the visibility of element names.
-@c ./../src/action.c 4039
+@c ./../src/action.c 3932
 
 If names are hidden you won't see them on the screen and they will not
 appear on the silk layer when you print the layout.
@@ -2043,7 +2025,7 @@ Undo()
 @end cartouche
 
 Undo recent changes.
-@c ./../src/action.c 5224
+@c ./../src/action.c 5117
 
 The unlimited undo feature of @code{Pcb} allows you to recover from
 most operations that materially affect you work.  Calling
@@ -2087,7 +2069,7 @@ Unselect(TextByName|ViaByName)
 @end cartouche
 
 unselects the object at the pointer location or the specified objects
-@c ./../src/action.c 4726
+@c ./../src/action.c 4619
 
 @table @code
 
@@ -2274,7 +2256,7 @@ About()@end format
 @end cartouche
 
 Tell the user about this version of PCB.
-@c ./../src/hid/gtk/gtkhid-main.c 1970
+@c ./../src/hid/gtk/gtkhid-main.c 1615
 
 This just pops up a dialog telling the user which version of
 @code{pcb} they're running.
@@ -2290,7 +2272,7 @@ AdjustStyle()
 @end cartouche
 
 Open the window which allows editing of the route styles
-@c ./../src/hid/gtk/gui-top-window.c 3827
+@c ./../src/hid/gtk/gui-top-window.c 3810
 
 Opens the window which allows editing of the route styles.
 
@@ -2305,7 +2287,7 @@ Center()
 @end cartouche
 
 Moves the pointer to the center of the window.
-@c ./../src/hid/gtk/gtkhid-main.c 2431
+@c ./../src/hid/gtk/gtkhid-main.c 2073
 
 Move the pointer to the center of the window, but only if it's
 currently within the window already.
@@ -2320,7 +2302,7 @@ Cursor(Type,DeltaUp,DeltaRight,Units)@end format
 @end cartouche
 
 Move the cursor.
-@c ./../src/hid/gtk/gtkhid-main.c 2530
+@c ./../src/hid/gtk/gtkhid-main.c 2172
 
 This action moves the mouse cursor.  Unlike other actions which take
 coordinates, this action's coordinates are always relative to the
@@ -2364,7 +2346,7 @@ DoWindows(Layout|Library|Log|Netlist|Preferences)@end format
 @end cartouche
 
 Open various GUI windows.
-@c ./../src/hid/gtk/gtkhid-main.c 2588
+@c ./../src/hid/gtk/gtkhid-main.c 2230
 
 @table @code
 
@@ -2402,7 +2384,7 @@ EditLayerGroups()
 @end cartouche
 
 Open the preferences window which allows editing of the layer groups
-@c ./../src/hid/gtk/gui-top-window.c 3850
+@c ./../src/hid/gtk/gui-top-window.c 3833
 
 Opens the preferences window which is where the layer groups
 are edited.  This action is primarily provides to provide menu
@@ -2418,7 +2400,7 @@ GetXY()@end format
 @end cartouche
 
 Get a coordinate.
-@c ./../src/hid/gtk/gtkhid-main.c 1985
+@c ./../src/hid/gtk/gtkhid-main.c 1630
 
 Prompts the user for a coordinate, if one is not already selected.
 
@@ -2434,7 +2416,7 @@ Popup(MenuName, [Button])@end format
 Bring up the popup menu specified by @code{MenuName}.
 If called by a mouse event then the mouse button number
 must be specified as the optional second argument.
-@c ./../src/hid/gtk/gtkhid-main.c 2655
+@c ./../src/hid/gtk/gtkhid-main.c 2297
 
 This just pops up the specified menu.  The menu must have been defined
 as a named subresource of the Popups resource in the menu resource
@@ -2451,7 +2433,7 @@ Print()@end format
 @end cartouche
 
 Print the layout.
-@c ./../src/hid/gtk/gtkhid-main.c 2303
+@c ./../src/hid/gtk/gtkhid-main.c 1948
 
 This will find the default printing HID, prompt the user for its
 options, and print the layout.
@@ -2466,7 +2448,7 @@ PrintCalibrate()@end format
 @end cartouche
 
 Calibrate the printer.
-@c ./../src/hid/gtk/gtkhid-main.c 2354
+@c ./../src/hid/gtk/gtkhid-main.c 1999
 
 This will print a calibration page, which you would measure and type
 the measurements in, so that future printouts will be more precise.
@@ -2484,7 +2466,7 @@ Save(PasteBuffer)@end format
 @end cartouche
 
 Save layout and/or element data to a user-selected file.
-@c ./../src/hid/gtk/gtkhid-main.c 2129
+@c ./../src/hid/gtk/gtkhid-main.c 1774
 
 This action is a GUI front-end to the core's @code{SaveTo} action
 (@pxref{SaveTo Action}).  If you happen to pass a filename, like
@@ -2502,7 +2484,7 @@ SelectLayer(1..MAXLAYER|Silk|Rats)@end format
 @end cartouche
 
 Select which layer is the current layer.
-@c ./../src/hid/gtk/gui-top-window.c 2856
+@c ./../src/hid/gtk/gui-top-window.c 2839
 
 The specified layer becomes the currently active layer.  It is made
 visible if it is not already visible
@@ -2517,7 +2499,7 @@ SetUnits(mm|mil)@end format
 @end cartouche
 
 Set the default measurement units.
-@c ./../src/hid/gtk/gtkhid-main.c 2627
+@c ./../src/hid/gtk/gtkhid-main.c 2269
 
 @table @code
 
@@ -2539,7 +2521,7 @@ SwapSides(|v|h|r)@end format
 @end cartouche
 
 Swaps the side of the board you're looking at.
-@c ./../src/hid/gtk/gtkhid-main.c 2211
+@c ./../src/hid/gtk/gtkhid-main.c 1856
 
 This action changes the way you view the board.
 
@@ -2582,7 +2564,7 @@ ToggleView(Silk|Rats|Pins|Vias|Mask|BackSide)@end format
 @end cartouche
 
 Toggle the visibility of the specified layer or layer group.
-@c ./../src/hid/gtk/gui-top-window.c 2780
+@c ./../src/hid/gtk/gui-top-window.c 2763
 
 If you pass an integer, that layer is specified by index (the first
 layer is @code{1}, etc).  If you pass a layer name, that layer is
@@ -2604,7 +2586,7 @@ Zoom(factor)@end format
 @end cartouche
 
 Various zoom factor changes.
-@c ./../src/hid/gtk/gtkhid-main.c 233
+@c ./../src/hid/gtk/gtkhid-main.c 176
 Changes the zoom (magnification) of the view of the board.  If no
 arguments are passed, the view is scaled such that the board just fits
 inside the visible window (i.e. ``view all'').  Otherwise,
@@ -2681,7 +2663,7 @@ About()@end format
 @end cartouche
 
 Tell the user about this version of PCB.
-@c ./../src/hid/lesstif/dialogs.c 870
+@c ./../src/hid/lesstif/dialogs.c 851
 
 This just pops up a dialog telling the user which version of
 @code{pcb} they're running.
@@ -2696,7 +2678,7 @@ AdjustSizes()@end format
 @end cartouche
 
 Let the user change the board size, DRC parameters, etc
-@c ./../src/hid/lesstif/dialogs.c 1154
+@c ./../src/hid/lesstif/dialogs.c 1135
 
 Displays a dialog box that lets the user change the board
 size, DRC parameters, and text scale.
@@ -2860,7 +2842,7 @@ DoWindows(Layout|Library|Log|Netlist)@end format
 @end cartouche
 
 Open various GUI windows.
-@c ./../src/hid/lesstif/dialogs.c 834
+@c ./../src/hid/lesstif/dialogs.c 815
 
 @table @code
 
@@ -2913,7 +2895,7 @@ EditLayerGroups()@end format
 @end cartouche
 
 Let the user change the layer groupings
-@c ./../src/hid/lesstif/dialogs.c 1466
+@c ./../src/hid/lesstif/dialogs.c 1447
 
 Displays a dialog that lets the user view and change the layer
 groupings.  Each layer (row) can be a member of any one layer group
@@ -2933,7 +2915,7 @@ Export()@end format
 @end cartouche
 
 Export the layout.
-@c ./../src/hid/lesstif/dialogs.c 960
+@c ./../src/hid/lesstif/dialogs.c 941
 
 Prompts the user for an exporter to use.  Then, prompts the user for
 that exporter's options, and exports the layout.
@@ -3021,7 +3003,7 @@ Print()@end format
 @end cartouche
 
 Print the layout.
-@c ./../src/hid/lesstif/dialogs.c 897
+@c ./../src/hid/lesstif/dialogs.c 878
 
 This will find the default printing HID, prompt the user for its
 options, and print the layout.
@@ -3036,7 +3018,7 @@ PrintCalibrate()@end format
 @end cartouche
 
 Calibrate the printer.
-@c ./../src/hid/lesstif/dialogs.c 940
+@c ./../src/hid/lesstif/dialogs.c 921
 
 This will print a calibration page, which you would measure and type
 the measurements in, so that future printouts will be more precise.
diff --git a/doc/pcbfile.texi b/doc/pcbfile.texi
index 70dc33a..7dd6be8 100644
--- a/doc/pcbfile.texi
+++ b/doc/pcbfile.texi
@@ -926,12 +926,4 @@ Move items orthogonally.
 Draw autoroute paths real-time.
 @item 0x20000
 New polygons are full ones.
-@item 0x40000
-Names are locked, the mouse cannot select them.
-@item 0x80000
-Everything but names are locked, the mouse cannot select anything else.
-@item 0x100000
-New polygons are full polygons.
-@item 0x200000
-When set, element names are not drawn.
 @end table
diff --git a/doc/version.texi b/doc/version.texi
index 0cf3402..bc949ab 100644
--- a/doc/version.texi
+++ b/doc/version.texi
@@ -1,4 +1,4 @@
-@set UPDATED 23 January 2009
-@set UPDATED-MONTH January 2009
+@set UPDATED 11 December 2007
+@set UPDATED-MONTH December 2007
 @set EDITION 1.99y
 @set VERSION 1.99y
